lib/msal-browser/src/crypto/CryptoOps.ts

```diff
@@ -203,8 +203,10 @@ export class CryptoOps implements ICrypto {
         // Generate header
         const header = {
             alg: publicKeyJwk.alg,
-            type: KEY_FORMAT_JWK
+            type: KEY_FORMAT_JWK,
+            kid: kid
```

Comment:
Any tests you add for this?

Reply:
The spec doesn't really require any of the header attributes as mandatory, so I'm not sure if there's much to test here. Open to suggestions though.

Reply:
I was about to comment the same on tests. Well, can we test if we are setting the header correctly if it is set indeed?